src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -108,43 +108,20 @@ export const TransactionGenericRow = ({
   } = GetTxLayoutFormats({ parentWidth })
 
   const transfers = useMemo(() => {
-    return displayTransfers.map((transfer, index) => (
-      <Stack
-        alignItems='center'
-        key={index}
-        flex={1}
-        mt={{ base: 2, md: 0, xl: compactMode ? 2 : 0 }}
-        direction={index === 0 ? 'row' : 'row-reverse'}
-        textAlign={index === 0 ? 'left' : 'right'}
-      >
-        <AssetIcon
-          assetId={transfer.asset?.assetId}
-          boxSize={{ base: '24px', lg: compactMode ? '24px' : '40px' }}
-        />
-        <Box flex={1}>
-          <Amount.Crypto
-            color='inherit'
-            fontWeight='medium'
-            value={fromBaseUnit(transfer.value, transfer.asset?.precision ?? FALLBACK_PRECISION)}
-            symbol={transfer.asset?.symbol ?? FALLBACK_SYMBOL}
-            maximumFractionDigits={4}
-          />
-          {transfer.marketData.price && (
-            <Amount.Fiat
-              color='gray.500'
-              fontSize='sm'
-              lineHeight='1'
-              value={bnOrZero(
-                fromBaseUnit(transfer.value, transfer.asset?.precision ?? FALLBACK_PRECISION),
-              )
-                .times(transfer.marketData.price)
-                .toString()}
-            />
-          )}
-        </Box>
-      </Stack>
-    ))
-  }, [compactMode, displayTransfers])
+    return displayTransfers.map((transfer, index) => {
+      const displayTransferType = transfer.type
```

Comment:
I know you didn't start it, but all of the "display transfer" naming feels off.

Because "display" is primarily used as a verb, but can also be an adjective (as it's been used here), it's both ambiguous and confusing. I look at `displayTransferType` and expect to see a function with a side effect that displays some text somewhere.

In this case, `displayedTransfers` would disambiguate and add clarity to the usage.

CC @kaladinlight.

Reply:
Absolutely agree the current verbage (which shouldn't really be a **verb**-age) is confusing unless you look at the actual consumption - stacked https://github.com/shapeshift/web/pull/3138 on top of this

Reply:
Yeah was never very happy with the name out of the gate. I am down with that update assuming nothing else makes more sense.

Reply:
With the changes brought in with this pr. I feel like we could also update `getDisplayTransfers` to something like `getTransfersByType` which reduces the transfers down into:
```
{
  "send": [...transfers],
  "receive": [...transfers],
}
```

and just call it `transfersByType`. Then it would be easy to `Object.entries()` over that payload, check if there are more than one transfer in the array and render accordingly. It would help disambiguate things and reduce an extra prop being passed in.

Reply:
Ohh very nice one, that's exactly what we needed here - 5e3e4d981750d8e856a3d219ec8ccafbb0b250ac